src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -23,6 +23,13 @@ import './app_container.scss';
 import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';
 import { IDataPluginServices } from '../../../data/public';
 import { QUERY_ENHANCEMENT_ENABLED_SETTING } from './constants';
+import {
+  DISCOVER_LOAD_EVENT,
+  NEW_DISCOVER_LOAD_EVENT,
+  NEW_DISCOVER_OPT_IN,
+  NEW_DISCOVER_OPT_OUT,
```

Comment:
Why do we need all 4? New discover cant be triggered without a page refresh. So the useEffect based tracking should be unnecessary.

Reply:
Oh. Removing the OPT_IN and OPT_OUT tracking from here.